### PR TITLE
Issue #7314 fixed.

### PR DESCRIPTION
**Pull request by suraj-webkul**
Redirect to review page on click on "Be the first to write a review" on product cards
#7314

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not working**, Please **compile** the **vue** file and push your code again.

**Comment by suraj-webkul** [comment]
Compiled vue, Please try again.

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue [hunk 1]
```diff
@@ -109,6 +111,10 @@
                 } else {
                     return false;
                 }
+            },
+
+            'redirectToReview': function() {
+                window.location.href = `product/${this.product.slug}/review`
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue:117]: add base url also,

this.$root.baseUrl

Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue:117]: https://github.com/bagisto/bagisto/pull/7316/commits/cd9d4ff7c8cd1610913b06db9b43685d38cb8553

Sir Already done using simple anchor tag

Comment (devansh-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue:117]: Then remove this method if not needed.

Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue]: I removed that method.

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is **working fine.**

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**